pwa-devdocs/pwa-buildpack/configuration-management.md

```diff
@@ -0,0 +1,187 @@
+# Configuration Management in Buildpack
```

Comment:
This file should be an `index.md` file and placed inside a `configuration-management.md` directory.

Also, each sentence should be on a separate line to make it easier to review and add targeted comments.

Alternatively, you can move it into the `_drafts` folder and I can continue the work of getting it published on the docs site in a separate ticket.

Reply:
Whoops, I'm sorry...I meant to put it in the drafts folder!